docs/modules/ROOT/pages/blocking-safe-by-default.adoc

```diff
@@ -52,20 +52,24 @@ __offload__ invocation of user code out of the event loop to a different thread.
 
 Although the outcome is simple that no user code should run on the event loop, implementations of this offloading are
 often either naive hence sub-optimal or fairly complex. ServiceTalk internally does the heavy lifting to make sure that
-it does not offload more than what is required. In other words, it reduces offloading when it can deduce that no user
+it does not offload more than what is required. In other words, it reduces offloading when it can determine that no user
 code can interact with the event loop on a certain path.
 
 === Execution Strategy
 
 The primary purpose of an link:{source-root}/servicetalk-transport-api/src/main/java/io/servicetalk/transport/api/ExecutionStrategy.java[Execution Strategy] is to define which interaction paths require offloading.
 
-At a general transport layer (no application level protocol), only two offload paths are available:
+For example, at the xref:{page-version}@servicetalk-http-api::blocking-safe-by-default.adoc[HTTP] transport layer
+(no application level protocol), four offload paths are available:
```

Comment:
HTTP is an application level protocol

Reply:
true, but that is not how application developers think of it. I will rephrase.